### PR TITLE
Add Project Affiliations Menu to Menu Bar

### PR DESCRIPTION
**Pull request by NadaElmasry**
## **Changes Implemented**

1. **User Class**
   - Updated the `User` class to retrieve associated projects using the user's ID. 
    
2. **Main Template**
     - Added a new **Project Affiliations** menu item to the main template.
     - Expanded the existing **Affiliations Menu** to include a **Project Affiliations** section.

3. **User Page Middleware**
     - Updated the user page middleware to incorporate the newly fetched project affiliations from the `User` class. 
     
## **Current Main Page View**
projectaffiliations1
projectaffiliations2

## **Related Issue**

- **Addresses #9279:** [\[Menu bar\] Add Project Affiliations](https://github.com/NadaElmasry/Loris/issues/9279)

### CODE REVIEW

**Comment by laemtl** [comment]
@NadaElmasry You introduced a lot of changes in the User classes . Some of them are the reason you have failed Unit tests.

**Comment by laemtl** [comment]
You have the following static errors:
FILE: /home/runner/work/Loris/Loris/php/libraries/User.class.inc
----------------------------------------------------------------------
FOUND 5 ERRORS AND 3 WARNINGS AFFECTING 8 LINES
----------------------------------------------------------------------
  36 | ERROR   | [x] Tag value for @var tag indented incorrectly;
     |         |     expected 4 spaces but found 1
  68 | ERROR   | [x] Equals sign not aligned with surrounding
     |         |     assignments; expected 3 spaces but found 1 space
 125 | ERROR   | [x] Equals sign not aligned with surrounding
     |         |     assignments; expected 6 spaces but found 1 space
 130 | ERROR   | [x] Equals sign not aligned with surrounding
     |         |     assignments; expected 2 spaces but found 1 space
 1[36](https://github.com/aces/Loris/pull/9405/checks#step:11:37) | ERROR   | [x] Equals sign not aligned with surrounding
     |         |     assignments; expected 3 spaces but found 1 space
 165 | WARNING | [ ] Line exceeds 85 characters; contains 115
     |         |     characters
 249 | WARNING | [ ] Line exceeds 85 characters; contains 106
     |         |     characters
 253 | WARNING | [ ] Line exceeds 85 characters; contains 98
     |         |     characters
----------------------------------------------------------------------
PHPCBF CAN FIX THE 5 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

**Comment by laemtl** [comment]
@NadaElmasry Can we remove all the User class changes except the ones needed for the feature?

File: php/libraries/User.class.inc [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 /**
- * This file contains the Loris User class
+ * This file contains the Loris user class
  *
- * PHP Version 7.4+
+ * PHP Version 7
  *
  * @category Main
  * @package  Main
+ * @author   Alex Zijdenbos <zijdenbos@example.com>
```
Comment (christinerogers) [inline php/libraries/User.class.inc:10]: changes like this (by your IDE) are cluttering -- we'd like to see just what you changed (as discussed Friday).   
@NadaElmasry if you need help with next steps here please let us know ASAP

**Pull request closed without merging**